Build issue on Archetype module

**Describe the bug**
The project isn't building because of onf an error on Archetype module.

**To Reproduce**
Steps to reproduce the behavior:
1.  mvn clean install
2. See attached logs.

**Expected behavior**
Project should build

**Desktop (please complete the following information):**
 - mvn -version output:
```
Apache Maven 3.6.2 (40f52333136460af0dc0d7232c0dc0bcf0d9e117; 2019-08-27T09:06:16-06:00)
Maven home: /maven/apache-maven-3.6.2
Java version: 1.8.0_212, vendor: AdoptOpenJDK, runtime: /Users/usertest/.sdkman/candidates/java/8.0.212.hs-adpt/jre
Default locale: en_GT, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
``` 
**Additional context**
Logs:
```
[INFO] [INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ myservice ---
[INFO] [INFO] No sources to compile
[INFO] [INFO]
[INFO] [INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ myservice ---
[INFO] [INFO] No tests to run.
[INFO] [INFO]
[INFO] [INFO] --- download-maven-plugin:1.4.0:wget (install-tracer) @ myservice ---
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time:  3.411 s
[INFO] [INFO] Finished at: 2019-10-07T09:21:19-06:00
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [ERROR] Failed to execute goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.0:wget (install-tracer) on project myservice: Execution install-tracer of goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.0:wget failed: java.lang.ClassNotFoundException: com.googlecode.download.maven.plugin.internal.CachedFileEntry -> [Help 1]
[INFO] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.0:wget (install-tracer) on project myservice: Execution install-tracer of goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.0:wget failed: java.lang.ClassNotFoundException: com.googlecode.download.maven.plugin.internal.CachedFileEntry
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
[INFO]     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
[INFO]     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
[INFO]     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
[INFO]     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
[INFO]     at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
[INFO]     at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
[INFO]     at java.lang.reflect.Method.invoke (Method.java:498)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] Caused by: org.apache.maven.plugin.PluginExecutionException: Execution install-tracer of goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.0:wget failed: java.lang.ClassNotFoundException: com.googlecode.download.maven.plugin.internal.CachedFileEntry
[INFO]     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:148)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
[INFO]     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
[INFO]     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
[INFO]     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
[INFO]     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
[INFO]     at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
[INFO]     at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
[INFO]     at java.lang.reflect.Method.invoke (Method.java:498)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] Caused by: java.lang.IllegalStateException: java.lang.ClassNotFoundException: com.googlecode.download.maven.plugin.internal.CachedFileEntry
[INFO]     at com.googlecode.download.maven.plugin.internal.cache.FileBackedIndex.loadFrom (FileBackedIndex.java:108)
[INFO]     at com.googlecode.download.maven.plugin.internal.cache.FileBackedIndex.<init> (FileBackedIndex.java:44)
[INFO]     at com.googlecode.download.maven.plugin.internal.cache.DownloadCache.<init> (DownloadCache.java:38)
[INFO]     at com.googlecode.download.maven.plugin.internal.WGet.execute (WGet.java:216)
[INFO]     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
[INFO]     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
[INFO]     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
[INFO]     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
[INFO]     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
[INFO]     at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
[INFO]     at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
[INFO]     at java.lang.reflect.Method.invoke (Method.java:498)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] Caused by: java.lang.ClassNotFoundException: com.googlecode.download.maven.plugin.internal.CachedFileEntry
[INFO]     at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass (SelfFirstStrategy.java:50)
[INFO]     at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass (ClassRealm.java:271)
[INFO]     at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:247)
[INFO]     at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:239)
[INFO]     at java.lang.Class.forName0 (Native Method)
[INFO]     at java.lang.Class.forName (Class.java:348)
[INFO]     at java.io.ObjectInputStream.resolveClass (ObjectInputStream.java:686)
[INFO]     at java.io.ObjectInputStream.readNonProxyDesc (ObjectInputStream.java:1868)
[INFO]     at java.io.ObjectInputStream.readClassDesc (ObjectInputStream.java:1751)
[INFO]     at java.io.ObjectInputStream.readOrdinaryObject (ObjectInputStream.java:2042)
[INFO]     at java.io.ObjectInputStream.readObject0 (ObjectInputStream.java:1573)
[INFO]     at java.io.ObjectInputStream.readObject (ObjectInputStream.java:431)
[INFO]     at java.util.HashMap.readObject (HashMap.java:1412)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
[INFO]     at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
[INFO]     at java.lang.reflect.Method.invoke (Method.java:498)
[INFO]     at java.io.ObjectStreamClass.invokeReadObject (ObjectStreamClass.java:1170)
[INFO]     at java.io.ObjectInputStream.readSerialData (ObjectInputStream.java:2178)
[INFO]     at java.io.ObjectInputStream.readOrdinaryObject (ObjectInputStream.java:2069)
[INFO]     at java.io.ObjectInputStream.readObject0 (ObjectInputStream.java:1573)
[INFO]     at java.io.ObjectInputStream.readObject (ObjectInputStream.java:431)
[INFO]     at com.googlecode.download.maven.plugin.internal.cache.FileBackedIndex.loadFrom (FileBackedIndex.java:106)
[INFO]     at com.googlecode.download.maven.plugin.internal.cache.FileBackedIndex.<init> (FileBackedIndex.java:44)
[INFO]     at com.googlecode.download.maven.plugin.internal.cache.DownloadCache.<init> (DownloadCache.java:38)
[INFO]     at com.googlecode.download.maven.plugin.internal.WGet.execute (WGet.java:216)
[INFO]     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
[INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
[INFO]     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
[INFO]     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
[INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
[INFO]     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
[INFO]     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
[INFO]     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
[INFO]     at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
[INFO]     at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
[INFO]     at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
[INFO]     at java.lang.reflect.Method.invoke (Method.java:498)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
[INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] [ERROR]
[INFO] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[INFO] [ERROR]
[INFO] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[INFO] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[INFO] Post-archetype-generation invoker exit code: 1
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for μGen :: Parent 0.1.0-SNAPSHOT:
[INFO]
[INFO] μGen :: Parent ..................................... SUCCESS [  0.764 s]
[INFO] μGen :: BOM ........................................ SUCCESS [  0.028 s]
[INFO] μGen :: Codegen .................................... SUCCESS [ 30.175 s]
[INFO] μGen :: Archetype .................................. FAILURE [  5.870 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  37.026 s
[INFO] Finished at: 2019-10-07T09:21:19-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:3.1.2:integration-test (default-integration-test) on project microgen-archetype:
[ERROR] Archetype IT 'openliberty' failed: Execution failure: exit code = 1
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :microgen-archetype
```

--- a/microgen-archetype/src/main/resources/archetype-resources/pom.xml
+++ b/microgen-archetype/src/main/resources/archetype-resources/pom.xml
@@ -85,7 +85,7 @@
             <plugin>
                 <groupId>com.googlecode.maven-download-plugin</groupId>
                 <artifactId>download-maven-plugin</artifactId>
-                <version>1.4.0</version>
+                <version>1.3.0</version>
                 <executions>
                     <execution>
                         <id>install-tracer</id>

--- a/microgen-archetype/src/test/resources/projects/openliberty/reference/pom.xml
+++ b/microgen-archetype/src/test/resources/projects/openliberty/reference/pom.xml
@@ -85,7 +85,7 @@
             <plugin>
                 <groupId>com.googlecode.maven-download-plugin</groupId>
                 <artifactId>download-maven-plugin</artifactId>
-                <version>1.4.0</version>
+                <version>1.3.0</version>
                 <executions>
                     <execution>
                         <id>install-tracer</id>

